src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -70,7 +70,7 @@ public void testComma() throws Exception {
         final String[] expected = {
             "39:17: " + getCheckMessage(MSG_LINE_PREVIOUS, ","),
         };
-        verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+        verify(checkConfig, getPath("InputSeparatorWrapAfter3904.java"), expected);
```

Comment:
Separate file for each test. Rename it to avoid misunderstanding. File name should contain short information about its purpose.